tools/ddprobe.cpp

```diff
@@ -69,8 +71,128 @@ syncThreadDesktop() {
   CloseDesktop(hDesk);
 }
 
+/**
+ * @brief Checks whether a given frame is entirely dark by evaluating the RGB values of each pixel.
+ *
+ * This function determines if the provided frame is completely dark by analyzing the RGB values of every pixel.
+ * It iterates over all pixels in the frame and compares each pixel's RGB channels to a defined darkness threshold.
+ * If any pixel's RGB values exceed this threshold, the function concludes that the frame is not entirely dark and returns `false`.
+ * If all pixels are below the threshold, indicating a completely dark frame, the function returns `true`.
```

Comment:
```suggestion
 * If all pixels are below the threshold, indicating a completely dark frame, the function returns `false`.
```

Also doesn't match the code.

Reply:
Whoops, forget to AI generate the docs after making the changes